src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -338,6 +339,16 @@ class ProductConfigFormController {
     })
   }
 
+  omitIrrelevantData (itemConfig) {
+    const data = this.productData.frequency === 'NA'
+      ? omit(itemConfig, ['RECURRING_START_MONTH', 'RECURRING_DAY_OF_MONTH', 'jcr-title', 'AMOUNT_WITH_FEES'])
```

Comment:
is ok 'jcr-title' lower-case?

Reply:
Yes, it comes into the object lowercase and never makes it to the API (now) which is what wants uppercase.